project-reports-lib.ts

```diff
@@ -75,6 +76,58 @@ export function getStringFromLabel(card: ProjectIssue, re: RegExp): string {
   return str
 }
 
+export function getLastCommentField(
+  issue: ProjectIssue,
+  field: string
+): string {
+  let val = ''
+
+  if (!issue.comments) {
+    return ''
+  }
+
+  for (let i = issue.comments.length - 1; i >= 0; i--) {
+    const comment = issue.comments[i]
+    if (!comment) {
+      break
+    }
+
+    const lines = comment.body.split(os.EOL)
+    for (const line of lines) {
+      const parts = line.trim().split(':')
+      if (parts.length === 2 && fuzzyMatch(parts[0], field)) {
+        val = parts[1].trim()
+        break
```

Comment:
I think you can just `return val` here instead of the two nested breaks to get out of the loops. At the end, just return `null`.